web/src/components/tooltips/TooltipWrapper.tsx

```diff
@@ -17,10 +18,27 @@ export default function TooltipWrapper(
   if (!tooltipContent) {
     return children;
   }
+  const [isOpen, setIsOpen] = useState(false);
+  const { isMobile } = useDimensions();
+
+  // Declare the event handlers outside of the JSX to avoid re-creating them on every render.
+  const handleMouseEnter = isMobile ? () => {} : () => setIsOpen(true);
+  const handleMouseLeave = isMobile ? () => {} : () => setIsOpen(false);
+  const handleClick = isMobile ? () => setIsOpen(!isOpen) : () => {};
+  const handleContentClick = isMobile ? () => setIsOpen(false) : () => {};
+  const handleContentPointerDownOutside = isMobile ? () => setIsOpen(false) : () => {};
```

Comment:
How about something like this to make it a bit more readable? :)

```suggestion
  const noop = () => {};
	const openTooltip = () => setIsOpen(true);
	const closeTooltip = () => setIsOpen(false);
	const toggleTooltip = () => setIsOpen(!isOpen);
	
  const handleMouseEnter = isMobile ? noop :  openTooltip;
  const handleMouseLeave = isMobile ? noop : closeTooltip;
  const handleClick = isMobile ? toggleTooltip : noop;
  const handleContentClick = isMobile ? closeTooltip : noop;
  const handleContentPointerDownOutside = isMobile ? closeTooltip : noop;
```

Reply:
Will do!